k8s/cmd/crossconnect-monitor/crossconnect_monitor.go

```diff
@@ -26,6 +26,8 @@ import (
 var closing = false
 var managers = map[string]string{}
 
+const NsmNamespaceEnv = "NSM_NAMESPACE"
```

Comment:
Can we have this constant somewhere in k8s/pkg and just refer to it here.
There are several other places where you use it so it makes sense to have it in a central place.
I will makre these for completenes.